src/Bicep.Core.Samples/Files/Completions/resourceObject.json

```diff
@@ -67,24 +67,6 @@
       "newText": "if (${1:condition}) {\n\t$0\n}"
     }
   },
-  {
```

Comment:
This is expected. There was a bug in SnippetsProvider. See corresponding change in SnippetsProvider.cs.